### DISCUSSION TITLE
不知道是不是bug

### DISCUSSION THREAD

**Discussion by gallantsa** [xiaojieonly/Ehviewer_CN_SXJ#1472] [Q&A]
一下载就显示ptoken，下载不了

**Comment by Truu70** [comment]
![Screenshot_20241027_154808_EhViewer](https://github.com/user-attachments/assets/1229a964-1fcd-4ad6-bce8-e7c35cc2b81d)

**Reply by gallantsa** [reply]
issues里已经有解决方法了
